libs/ui/src/admin/ArtApprovalModel/ArtApprovalForm.tsx

```diff
@@ -0,0 +1,243 @@
+  VStack,
+  Image,
+  Center,
+  Flex,
+} from '@chakra-ui/react'
+import { Splide, SplideSlide } from '@splidejs/react-splide'
+import { BiEditAlt } from 'react-icons/bi'
+import { HiDotsVertical, HiOutlineCheck, HiOutlineX } from 'react-icons/hi'
+
+import { ArtImageProps, ArtApprovalFormTypes } from './types'
+
+const ArtImage: FC<ArtImageProps> = memo(({ image, alt }) => {
+  return (
+    <Image
+      objectFit="contain"
+      src={`${process.env['NX_API_URL']}${image.url}`}
+      alt={alt}
+    />
+  )
+})
+export const ArtApprovalForm: FC<ArtApprovalFormTypes> = ({
+  onReject,
+  onApprove,
+  onDelete,
+  onEdit,
+  art,
+  user,
+}) => {
+  // onReject and onApprove should receive art id, feedback, editor id
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  const handleSizeClick = newSize => {
+    onOpen()
+  }
+
+  const handleReject = data => {
+    onReject(data)
+  }
+  const handleEdit = data => {
+    onEdit(data)
+  }
+  const handleApprove = data => {
+    onApprove(data)
+  }
+  const handleDelete = data => {
+    onDelete(data)
+  }
+  const handleInputChange = data => {
+    console.log(data)
+  }
+  return (
+    <>
+      <Button
+        onClick={() => handleSizeClick('full')}
+        m={4}
+      >{`Open ArtApproval Modal`}</Button>
+      <Modal
+        onClose={onClose}
+        // size={['xs', 'sm', 'md', 'lg', 'xl', 'full']}
+        size={['1088px']}
+        isOpen={isOpen}
+      >
+        <ModalOverlay />
+        <ModalContent>
+          <ModalCloseButton />
+          <ModalBody>
+            <Stack direction={['column', 'row']} spacing={4}>
+              <Stack w={'full'}>
+                {art.images && art.images.length > 1 ? (
+                  <Splide>
+                    {art?.images.map(image => (
+                      <Center as={SplideSlide} key={image.id}>
+                        <ArtImage image={image} alt={art.title} />
+                      </Center>
+                    ))}
+                  </Splide>
+                ) : (
+                  <ArtImage image={art.images?.[0]} alt={art.title} />
+                )}
+              </Stack>
+              <VStack spacing={4} align={'start'} justify="space-between">
+                <VStack align={'start'} justify="space-between">
+                  <Flex align="start" justify="start" w="full" h={8}>
+                    <Text color={'blue.400'} fontWeight={'bold'}>
+                      {art?.title}
+                    </Text>
+                  </Flex>
+                  <HStack spacing={3} w={'full'}>
+                    {/* TODO art owner avatar should be here*/}
+                    <Avatar
+                      w={8}
+                      h={8}
+                      src="https://bit.ly/dan-abramov"
+                    ></Avatar>
+                    <Text fontWeight={'bold'}>{art?.artist?.name}</Text>
+                  </HStack>
+                  <Flex align="start" justify={'start'} w="full" h={'72px'}>

```

Comment:
- [ ] `maxH` is better in case art has a short description.
- [ ] You could also show long and short description cases in storybook

```suggestion
                  <Flex align="start" justify={'start'} w="full" maxH={'72px'} overflow='auto'>
```